dashboard/app/models/peer_review.rb

```diff
@@ -133,4 +139,28 @@ def self.create_for_submission(user_level, level_source_id)
       end
     end
   end
+
+  def self.get_peer_review_summaries(user, script)
+    peer_review_summaries = []
+
+    if user &&
+        script.professional_learning_course? &&
+        Plc::EnrollmentUnitAssignment.exists?(user: user, plc_course_unit: script.plc_course_unit)
+
+      peer_review_summaries = PeerReview.where(reviewer: user, script: script).map(&:summarize)
```

Comment:
I don't think `peer_review_summaries` is needed — can just `return PeerReview.where...` here.